cime/config/e3sm/machines/config_compilers.xml

```diff
@@ -1410,8 +1411,7 @@ for mct, etc.
 </compiler>
 
 <compiler COMPILER="gnu" MACH="snl-white">
-  <KOKKOS_PATH>$ENV{KOKKOS_ROOT}</KOKKOS_PATH>
-  <MPICXX>${KOKKOS_ROOT}/bin/nvcc_wrapper</MPICXX>
+  <MPICXX>${SHAREDLIBROOT}/bin/nvcc_wrapper</MPICXX>
   <CXXFLAGS>
     <append>-expt-extended-lambda -DCUDA_BUILD</append>
```

Comment:
Kokkos has `default_arch=sm_35` in nvcc_wrapper, which works for white. However, it's a good practice to specify `-arch=xyz` in the cxx options, since it's likely people will copy and paste this snippet for other gpu machines. They will still have to specify the correct sm, but at least it's a detail easier to detect. Also, if Kokkos changes the default to sm_70 (V100), then we may have problems, unless we specify a different one.